prometheus_hardware_exporter/collectors/ipmi_dcmi.py

```diff
@@ -0,0 +1,37 @@
+"""IPMI DCMI metrics collector."""
+
+import re
+from logging import getLogger
+from typing import Dict
+
+from ..utils import Command
+
+logger = getLogger(__name__)
+
+CURRENT_POWER_REGEX = re.compile(r"^Current Power\s*:\s*(?P<value>[0-9.]*)\s*Watts.*")
+
+
+class IpmiDcmi(Command):
+    """Command line tool for ipmi dcmi."""
+
+    prefix = "sudo"
```

Comment:
There's no need to prefix `sudo`. The package is expected to be used as root user. We should probably warn user that they should run as root somehow

Reply:
Discuss:
I have different opinion for this one. We can't expected user always run it as root user. The exporter should be workable for any user which has sudo permission.
The exporter itself should not relate on charm or snap.

Reply:
I think it's the same case as `sosreport`. It will failed if the user is not running it as `root` or with `sudo`. This is an adminstration tool and should be only used by administrator.